javav2/example_code/ec2/src/main/java/com/example/ec2/DescribeSecurityGroups.java

```diff
@@ -1,25 +1,15 @@
-//snippet-sourcedescription:[DescribeSecurityGroups.java demonstrates how to get information about all the Amazon EC2 security groups.]
-//snippet-keyword:[SDK for Java 2.0]
+//snippet-sourcedescription:[DescribeSecurityGroups.java demonstrates how to get information about all the Amazon Elastic Compute Cloud (Amazon EC2) security groups.]

```

Comment:
all of the (everywhere)

Reply:
Updated example to use upper case for Security Group